client/tx/aux_builder.go

```diff
@@ -190,16 +212,43 @@ func (b *AuxTxBuilder) GetSignBytes() ([]byte, error) {
 		}
 	case signing.SignMode_SIGN_MODE_LEGACY_AMINO_JSON:
 		{
-			signBz = legacytx.StdSignBytes(
```

Comment:
The key change in this PR, StdSignBytes calls [legacyMsg.GetSignBytes](https://github.com/cosmos/cosmos-sdk/blob/4a6a1e3cb8de459891cb0495052589673d14ef51/x/auth/migrations/legacytx/stdsign.go#L56), so we're now using the amino json signer in x/tx instead.